lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.spec.ts

```diff
@@ -294,4 +294,27 @@ describe('PeopleCloudComponent', () => {
             expect(checkUserHasRoleSpy).not.toHaveBeenCalled();
         });
     }));
+
+    it('should not validate preselect values if preselectValidation flag is set to false', () => {
```

Comment:
can we have 3 different unit tests
1 validate true and preSelectUser with id
2 validate true and preSelectUser with email
3 validate true and preSelectUser with username

Reply:
Can we simulate the case a user is not valid that the warning event is correctly emitted ?